src/Command/Site/AbstractCommand.php

```diff
@@ -356,13 +346,18 @@ protected function hasSiteRoot() {
   protected function siteConfig(InputInterface $input) {
     $siteName = $input->getArgument('name');
 
-    $config = $this->configurationManager->readTarget($siteName . '.' . $this->getEnv());
-
+    // Support for site name without env or with env.
+    $config = $this->configurationManager->readTarget($siteName);
+    if (empty($config))
+    {
+      $config = $this->configurationManager->readTarget($siteName . '.' . $this->getEnv());
+    }
     if (empty($config))
     {
       $message = sprintf(
-        'Site not found on %s env. To see a list of available sites, run drupal site:debug',
-         $this->getEnv()
+        'Site %s not found on %s env. To see a list of available sites, run drupal site:debug',
```

Comment:
This needs to be changed to be 'drupal debug:site'

Reply:
Yes